geoprocessing/src/modules/scenario-planning-units/dto/attach.planning-units.scenario.dto.ts

```diff
@@ -0,0 +1,40 @@
+import {
+  IsOptional,
+  IsEnum,
+  IsUUID,
+  Min,
+  Max,
+  IsInt,
+  IsArray,
+} from 'class-validator';
+import { PlanningUnitsJob } from 'src/modules/planning-units/dto/create.regular.planning-units.dto';
+import { IUCNCategory } from 'src/modules/protected-areas/protected-areas.geo.entity';
+
+/**
+ * @description
+ * With the scenario, we can easily retrieve the pus so we can attache them to the scenario
+ * Also if wdpaThreshold is in the data shared with the job we should concatenate it with the
+ * calculation of level of protection of the PUs
+ * @todo
+ * We have this dto partially duplicated in the api service in
+ * @file api/src/modules/scenarios/dto/create.scenario.dto.ts
+ */
+export class PlanningUnitsForScenarioJob extends PlanningUnitsJob {
```

Comment:
Where `PlanningUnitsJob` properties values are coming from? (not directly related to this PR, rather just wanted to know where to get them from - for example, `adminAreaLevel2Id` is available within all of project/scenario/user)